### PR TITLE
Clarification on permisions for the Redis socket

### PR DESCRIPTION
**Pull request by FalkenStein42**
After running into the problem myself and searching online for a solution i think a clarification on how to ensure proper permissions on the redis socket would be helpful to novice administrators

 Signed-off-by: Sergio Solé Rodríguez <sergio@sergiosole.dev>

### CODE REVIEW

**Review by tflidd — COMMENTED** [review]
You need to restart `redis` and `redis-server`?

**Comment by FalkenStein42** [comment]
I am sure redis-server needs to be restarted in order to regenerate the redis socket file, whether you have to restart the client as well i am unsure as i restarted everything once i applied the configuration

**Comment by tflidd** [comment]
In some oler tutorial that included redis, it just restarts `redis-server` (https://bayton.org/docs/nextcloud/installing-nextcloud-on-ubuntu-16-04-lts-with-redis-apcu-ssl-apache/). However, it says that you need to add redis to the services to be started at startup whereas the current Nextcloud documentation says this is automatically done. Not sure if you can easily verify this? I currently don't have a native Linux setup at hand to test it...

File: admin_manual/configuration_server/caching_configuration.rst [hunk 1]
```diff
@@ -124,9 +124,15 @@ to the redis group::
...
-You might need to restart apache for the changes to take effect::
+And modify the ``unixsocketperm`` of the ``redis.conf`` accordingly::
+
+  unixsocketperm 770
+
+You might need to restart apache and redis for the changes to take effect::
 
  systemctl restart apache2
+ systemctl restart redis-server
+ systemctl restart redis
```
Comment (tflidd) [inline admin_manual/configuration_server/caching_configuration.rst:135]: I just installed `redis-server` on ubuntu. In this case, you have `redis-server` as service. `redis` in this case is just an alias and this way you would restart redis twice:
image

Let's just keep the redis-server.

**Review by tflidd — CHANGES_REQUESTED** [review]

**Comment by FalkenStein42** [comment]
I have come acros a similar tutorial that just said 'restart redis' or something like that but didnt give a specific answer either. I could set up a quick vm to test it this week and maybe make a more comprehensive exlplanation of other parts.
Id prefer if someone familiar with redis / php cache in general help me with configuration as i am fairly new to it.
I could just test only wich components need to be restarted though.

Sent with [ProtonMail](https://protonmail.com/) secure email.

------- Original Message -------

File: admin_manual/configuration_server/caching_configuration.rst (hunk 1 above)
Comment (kesselb) [inline admin_manual/configuration_server/caching_configuration.rst:135]:
```suggestion
```

**Review by kesselb — APPROVED** [review]
Thanks :+1:

**Pull request merged**